editor/src/document/document_file.rs

```diff
@@ -601,11 +601,14 @@ impl MessageHandler<DocumentMessage, &InputPreprocessor> for DocumentMessageHand
 					self.layer_range_selection_reference = selected;
 				}
 
-				// Add or set our selected layers
-				if ctrl {
-					responses.push_front(AddSelectedLayers(paths).into());
-				} else {
-					responses.push_front(SetSelectedLayers(paths).into());
+				// Don't create messages for empty operations
```

Comment:
I realized, based on your astute suggestion @TrueDoctor, to monitor the messages with '3' + log,  that we were sending empty messages on occasion. Resolved that.